### DISCUSSION TITLE
free style libre 3 and web follower

### DISCUSSION THREAD

**Discussion by fabricejoubert** [NightscoutFoundation/xDrip#2201] [Q&A]
Hello,
I've been using the "web follower" feature for a few days now to retrieve the blood sugar values ​​from free style libre 3 ( throught abbot api). It works  well, but over this period I have a value every 10 minutes, on others every 5 minutes. Do you know the reason? Can we fix it? 10 minutes is sometimes a bit long.... thank you in advance.

**Comment by TobiasB91** [comment]
I have the same problem. Did you find a fix for your problem?

**Reply by fabricejoubert** [reply]
Hello,
No I am still using "web follower" with the same problem. I saw that there was a new solution to recover data from fsl3 with juggluco and a patched abbot application. But 3 apps (juggluco, xdrip and the patched abbot app), I think that's a lot... Has anyone tried it? Is it stable?

**Comment by ikimi66** [comment]
i, there, Does Anyone know why my xdrip+ could not use web follower to get libre3 librelinkup data, unavailable from the day before yesterday (10/26). The log shows collector service due to lack of reception: backoff:120. What's the problem?
I've also updated the latest xdrip+ still get the same issue.
![199387289-7a2059dd-f538-4956-aedb-1ade687df920](https://user-images.githubusercontent.com/103375139/200716424-da8af2cf-aa5e-4482-939b-669f77931832.jpg)
